src/DynamoCoreWpf/Views/Search/LibraryView.xaml

```diff
@@ -603,10 +604,10 @@
                     <Button.ContextMenu>
                         <ContextMenu Style="{StaticResource AddButtonContextMenu}">
                             <MenuItem Focusable="False"
-                                      Header="Import Library..."
+                                      Header="{x:Static p:Resources.DynamoViewFileMenuImport}"
```

Comment:
I hope DynamoViewFileMenuImport is already defined in resource file, or did you miss to include resource file here?